communication/state.hpp

```diff
@@ -280,7 +280,7 @@ State::conf_t State::default_config()
     conf.mav_state               = MAV_STATE_BOOT;
     conf.simulation_mode         = HIL_OFF;
     conf.autopilot_type          = MAV_TYPE_QUADROTOR;
-    conf.autopilot_name          = MAV_AUTOPILOT_MAVRIC;
```

Comment:
don't you want to comment that line with a TODO mark instead or deleting

I guess the mid term goal would be to become part of mavlink known autopilot, isn't it ?